apps/common/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class UserResource(models.Model):
```

Comment:
Let's change this to Resource

Also, add UserResource and use if a user mutate any idea in the future.. make sure to add created_by and modifed_by also